server/controller/restaurantController.js

```diff
@@ -1,66 +1,76 @@
-const axios = require('axios');
-const Restaurant = require('../models/Restaurant');
-const { testRestaurant, testRestaurants } = require('../seeds/testSeeds');
-const testAPI = require('../index');
+const axios = require("axios");
+const Restaurant = require("../models/Restaurant");
+const { testRestaurant, testRestaurants } = require("../seeds/testSeeds");
+const testAPI = require("../index");
 
 const restaurantsController = {
   getRestaurants: async function (req, res) {
     if (testAPI) {
-      res.json(testRestaurants)
+      res.json(testRestaurants);
     } else {
-      const latitude = req.body.latitude
-      const longitude = req.body.longitude
-      const type = '$type=restaurant'
+      const latitude = req.body.latitude;
+      const longitude = req.body.longitude;
+      const type = "$type=restaurant";
       let radius, keyword, max_price, min_price, open_now;
-      req.body.radius ? radius = `&radius=${req.body.radius}` : radius = ''
-      req.body.keyword ? keyword = `&keyword=${req.body.keyword}` : keyword = ''
-      req.body.max_price ? max_price = `&maxprice=${req.body.max_price}` : max_price = ''
-      req.body.min_price ? min_price = `&minprice=${req.body.min_price}` : min_price = ''
-      req.body.open_now ? open_now = `&opennow=${req.body.open_now}` : open_now = ''
-      const apiKey = process.env.GOOGLE_PLACES_APIKEY
+      req.body.radius ? (radius = `&radius=${req.body.radius}`) : (radius = "");
+      req.body.keyword
+        ? (keyword = `&keyword=${req.body.keyword}`)
+        : (keyword = "");
+      req.body.max_price
+        ? (max_price = `&maxprice=${req.body.max_price}`)
+        : (max_price = "");
+      req.body.min_price
+        ? (min_price = `&minprice=${req.body.min_price}`)
+        : (min_price = "");
+      req.body.open_now
+        ? (open_now = `&opennow=${req.body.open_now}`)
+        : (open_now = "");
+      const apiKey = process.env.GOOGLE_PLACES_APIKEY;
```

Comment:
You might run into merge conflicts with this when you try to merge with the `develop` branch since there were some minor changes to it, let me know if you need help or if you get confused when doing so.